indico/modules/events/papers/templates/management/competences.html

```diff
@@ -0,0 +1,34 @@
+{% from 'forms/_form.html' import form_header, form_field, form_footer %}
+{% extends 'layout/conference_page_base.html' %}
+
+{% block description %}
+    {%- trans -%}
+        Defining competences to the members of the teams may be helpful for the managers and judges during the paper assignment process.
+    {%- endtrans -%}
+{%- endblock %}
+
+{% block content -%}
+    <div id="competences-dialog-content">
+        {{ form_header(form) }}
+            <table class="i-table-widget">
+                <thead>
+                    <tr>
+                        <th class="name-column">{% trans %}Name{% endtrans %}</th>
+                        <th>{% trans %}Competences{% endtrans %}</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for entry in event.cfp.acl_entries %}
+                        <tr>
+                            <td class="icon-user competences-user">{{ entry.full_name }}</td>
+                            <td>{{ form['competences_{}'.format(entry.id)] }}</td>
+                        </tr>
+                    {% endfor %}
+                </tbody>
+            </table>
+        {% call form_footer(form) %}
+            <input class="i-button big highlight" type="submit" value="{% trans %}Save{% endtrans %}">
+            <a href="{{ back_url }}" class="i-button big" data-button-back>{% trans %}Cancel{% endtrans %}</a>
```

Comment:
you don't have a `back_url` - check the `simple_form` macro, i'm quite sure we omit it and/or use a `<button>` in case of no `back_url`